.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```diff
@@ -0,0 +1,50 @@
+name: Feature Request
+description: Suggest a new feature or improvement for the Flutter app using Firebase.
+labels:
+  - "enhancement"
+body:
+  - type: textarea
+    attributes:
+      label: Is your feature request related to a problem? Please describe.
+      description: >
+        A clear and concise description of what the problem is.  
+        Example: "I'm always frustrated when [...]"
+    validations:
+      required: true
+  - type: textarea
```

Comment:
Add one more dropdown with.
"Would you like to work on this issue"
"yes" or "no"